openeogeotrellis/geopysparkdatacube.py

```diff
@@ -945,7 +945,7 @@ def merge_cubes(self, other: 'GeopysparkDataCube', overlaps_resolver:str=None):
                     + f" Left names: {leftBandNames}, right names: {rightBandNames}.",
                 )
 
-        # TODO properly combine bbox and temporal extents in metadata?
+        # TODO properly combine bbox and temporal extents in metadata? Yes! Then pass to _apply_to_levels_geotrellis_rdd
```

Comment:
No need to pass to `_apply_to_levels_geotrellis_rdd`.
Combining temporal extent has been covered.